Community-Committee-Charter.md

```diff
@@ -73,6 +73,12 @@ Multiple-candidate methods may be reduced to simple election by plurality when t
 
 The CommComm will elect from amongst voting CommComm members a CommComm Chairperson to work on building an agenda for CommComm meetings and collaborate with the Individual Membership Directors the wishes of the CommComm to the Board for a term of one year according to the Node.js Foundation’s By-laws. The CommComm shall hold annual elections to select a CommComm Chairperson; there are no limits on the number of terms a CommComm Chairperson may serve.
 
+The CommComm will elect from amongst voting CommComm members: 
+  - a CommComm Chairperson, responsible for preparing agendas and coordinating CommComm meetings, and 
+  - a representative to serve on the Node.js Foundation Board of Directors pursuant to its Bylaws. 
+Each person shall serve a one year term, and may serve for more than one term. The
+same person may not hold both positions at the same time.
```

Comment:
Update wording: "may not hold" -> "should not hold" ?
If a person can hold both positions at the same time than it's fine.